### ISSUE TITLE
Create global js

### ISSUE THREAD

**Issue by kostyrenko-si** [mailru/jira-plugins-jsincluder#18]
Please, add in plagin possibility usingglobal js

**Comment by dariarostshina** [comment]
Hello, global js you can add in banner

**Comment by RJanata** [comment]
True, it can be done through Announcement Banner but thanks to JsIncluder you can split many script into section and have them organized in better way.
